strongbox-parent/pom.xml

```diff
@@ -660,6 +660,12 @@
                 <version>${version.logback}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.codehaus.janino</groupId>
+                <artifactId>janino</artifactId>
```

Comment:
What is this dependency for and under what license is it?

Reply:
it's under BSD

https://en.wikipedia.org/wiki/BSD_licenses

Reply:
used [here](https://github.com/strongbox/strongbox/pull/565/files#diff-846a8c163fa25f137004008783538413R14)